test/test_code_generation.py

```diff
@@ -50,23 +52,31 @@ def test_generate_weblab_model(tmp_path):
     model = os.path.join('test', 'models', 'hodgkin_huxley_squid_axon_model_1952_modified.cellml')
     model = cellmlmanip.load_model(model)
 
-    # Select model outputs (as qualified names)
-    oxmeta = 'https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata#'
+    # Select model outputs
     outputs = [
-        (oxmeta, 'membrane_fast_sodium_current'),
-        (oxmeta, 'membrane_voltage'),
-        (oxmeta, 'time'),
-        (oxmeta, 'state_variable'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'membrane_fast_sodium_current'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'membrane_voltage'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'time'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'state_variable'),
     ]
 
-    # Select model parameters (as qualified names)
+    # Select model parameters
     parameters = [
-        (oxmeta, 'membrane_fast_sodium_current_conductance'),
-        (oxmeta, 'membrane_potassium_current_conductance'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'membrane_fast_sodium_current_conductance'),
+        VariableReference.create('oxmeta', OXMETA_NS, 'membrane_potassium_current_conductance'),
     ]
 
+    # Annotate state variables with the magic oxmeta:state_variable term
```

Comment:
Why are we doing this? Just to test that it can deal with them already being present? Should we have a test for when they're not present or partially present?

Reply:
It's needed here because it's this annotation that create_weblab_model now uses to build the state variable vector, so without it the vector would be empty.

Reply:
So will we need to add it in any models we use? Otherwise not sure why it's not inside of create_weblab_model ?

Reply:
The `modify_model` logic sets it normally; it's only here in the test because that machinery isn't being used. No need to add it to models. It's not inside `create_weblab_model` because setting it correctly needs info that function doesn't have, so that e.g. state variables that are units-converted as outputs are reported correctly in the `get_outputs` method of the generated model.